macros_doc.md

```diff
@@ -26,7 +76,11 @@ you to use macros. Even adding a table may not require you to use macros unless
 the table falls into one of the categories listed in the macros table.
 
 If you want to add content with a macro and need help, do not hesitate to
-contact a member of the bids-maintainers for help.
+contact a member of the bids-maintainers for help. To do this, you can either
+mention an individual maintainer by their GitHub username or mention the whole
+team (`@bids-standard/maintainers`).
+
+## Do I need learn how to code in Python to use those macros?
```

Comment:
This is a rewrite of lines 83-96

## Do I need to know how to program to use macros?

Macros don't require programming knowledge to use.  You do need to know what arguments the macro expects. The examples linked in the above table provide useful guidance in this respect.

Macros that extract information from the schema also require you to use the correct terms in the schema. This process is illustrated in the next section.

Note that under the hood the macros themselves call python code that can be found in the [`tools` folder](https://github.com/bids-standard/bids-specification/tree/master/tools). If you are interested in creating a new macro for users, this would be useful.